www-servers/nginx/nginx-1.19.7.recipe

```diff
@@ -11,10 +11,11 @@ COPYRIGHT="2002-2019 Igor Sysoev
 LICENSE="BSD (2-clause)"
 REVISION="2"
 SOURCE_URI="https://nginx.org/download/nginx-$portVersion.tar.gz"
-CHECKSUM_SHA256="5e333687464e1d6dfb86fc22d653b99a6798dda40093b33186eeeec5a97e69ec"
+CHECKSUM_SHA256="7ae4dd020c41d3a5e1e6a8578fcc60e508e3e27e7668e845ddc87a05a775b50e"
 PATCHES="nginx-$portVersion.patchset"
+ADDITIONAL_FILES="userlaunch.in"
 
-ARCHITECTURES="?x86_gcc2 ?x86 ?x86_64"
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
```

Comment:
This can be build for secondary architecture (32bit) with:
```diff
+SECONDARY_ARCHITECTURES="x86"
```
Not sure how to use this, but at least I had to create a user and group "nobody" to have it started from Terminal, also had to manually create the directory /system/var/nginx/logs ...

Reply:
I'll add it x86. I haven't had a lot of luck getting a 32bit VM to spin up.

The default install requires some configuration before you can launch it. The default configuration has the parent process run as "user" and launch child processes as "nobody". As the default Haiku install doesn't have "nobody", it has to be created or the configuration changed. I'll figure something up to make this more seamless.

For the logs folder, it appears that I need to do like the lighttpd port and have a post-install script create the folders (several in $localStateDir). I'll work on this some this weekend.

Reply:
For the post-install script only "/system/var/nginx/logs" is needed, the other ones are created by nginx
For the configuration, had played a bit with it (like "usr nobody nogroup" and some things I found on the net) only thing that worked for as far as I checked was groupadd and useradd, thanks for looking into this!

Reply:
I can't figure out why the folder isn't already there. It's in the built package file (when I expand it).

I think I'm missing something in how packages work, but I've only been poking around with Haiku for maybe 2 weeks now so I've got a bit of learning left to do.